lib/slackify/handlers/validator.rb

```diff
@@ -19,10 +19,28 @@ def self.verify_handler_integrity(handler)
         handler.dig(handler_name, 'commands').each do |command|
           command_errors = []
 
-          unless command['regex'].is_a?(Regexp)
+          unless command['regex'] || command['base_command']
+            command_errors.append('No regex or base command was provided.')
+          end
+
+          if command['regex'].present?
+            if command['base_command'].present?
+              command_errors.append('Regex and base_command cannot be used in the same handler.')
+            end
+
+            if command['parameters'].present?
+              command_errors.append('Regex and parameters cannot be used in the same handler.')
+            end
+          end
+
+          unless command['regex'].nil? || command['regex'].is_a?(Regexp)
             command_errors.append('No regex was provided.')
           end
 
+          unless command['base_command'].nil? || command['base_command'].is_a?(String)
```

Comment:
Same thing here

Reply:
This is a bit more nuanced, Rails would make it cleared as `unless command['base_command'].present? && command['base_command'].is_a?(String)`

if it's present and its not a string.

I've switched it to `if command['base_command'] && !command['base_command'].is_a?(String)` for now, but I want to add a better validation to confirm this method exists in a future PR